next.config.js

```diff
@@ -13,6 +13,7 @@ const env = {
   ENVIRONMENT_NAME: process.env.ENVIRONMENT_NAME,
   PROCESS_NAME: process.env.PROCESS_NAME,
   WORKTRAY_URL: process.env.WORKTRAY_URL,
+  TENANCY_URL: process.env.TENANCY_URL,
```

Comment:
This will need adding to CI config, too. Rebase on top of the latest `development` and you should see the build fail.